graphix/command.py

```diff
@@ -51,7 +53,30 @@ class M(Command):
     angle: float = 0.0
     s_domain: list[Node] = []
     t_domain: list[Node] = []
-    vop: int = 0
+
+    def clifford(self, clifford: Clifford) -> M:
+        s_domain = self.s_domain
+        t_domain = self.t_domain
+        for gate in clifford.hsz:
+            if gate == graphix.clifford.I:
+                pass
+            elif gate == graphix.clifford.H:
+                t_domain, s_domain = s_domain, t_domain
+            elif gate == graphix.clifford.S:
+                t_domain = s_domain + t_domain
+            elif gate == graphix.clifford.Z:
+                pass
+            else:
+                assert False
```

Comment:
`assert` can be bypassed. Please raise `RuntimeError` instead.

Reply:
Fixed in adc6209.